.github/workflows/test.yml

```diff
@@ -169,33 +129,27 @@ jobs:
         run: |
           gem install jekyll -v 4.2.0
           gem install jekyll-redirect-from
-      - name: Download all built API docs
-        uses: actions/download-artifact@v2
-        with:
-          path: artifacts
-      - name: Untar build artifacts
-        run: for f in $(find artifacts -name "*.tar.gz"); do tar zxf "$f"; done
       - name: Install CIRCT
         uses: ./.github/workflows/install-circt
       - name: Build the microsite
         run: make -C website
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: Deploy Website to GitHub Pages (From Master Branch)
-        if: (github.event_name == 'push') && (github.ref_type == 'branch') && (github.ref_name == 'master')
-        uses: JamesIves/github-pages-deploy-action@3.7.1
+      - name: Tar built website
+        run: tar zcf website.tar.gz website/docs/target/site
+      - name: Share Built Website
+        uses: actions/upload-artifact@v2
```

Comment:
```suggestion
        uses: actions/upload-artifact@v3
```

Reply:
what does this mean? Is it a new artifact or just a new version of the action?